docs/howto/save_buckets_to_file.ipynb

```diff
@@ -4,15 +4,26 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Save buckets to files\n",
+    "# Saving bucket information to a file\n",
     "\n",
-    "After the bucketing process has been completed, it's useful to save the buckets to a file.<br>\n",
-    "`skorecard` supports saving bucketers to yaml and json files."
+    "If you have a specific set of bucketing boundaries you are satisfied with, it's useful to save them to a file.\n",
+    "You might want to save the bucketing information a configuration files along with your code.\n",
```

Comment:
'and' configuration files

Reply:
'as' configuration files actually. Typo fixed thnx